mimic/rest/identity_api.py

```diff
@@ -294,13 +294,26 @@ def validate_token(self, request, token_id):
         """
         Creates a new session for the given tenant_id and token_id
         and always returns response code 200.
-        Docs: http://developer.openstack.org/api-ref-identity-v2.html#admin-tokens
+        Docs: http://developer.openstack.org/api-ref-identity-admin-v2.html#
```

Comment:
Not really, but there's no way to break a URI that I'm aware of and the linter will otherwise complain. Open to suggestions.